app/components/pools/pool-new.ts

```diff
@@ -0,0 +1,90 @@
+import {ClusterService} from '../rest/clusters';
+import {VolumeHelpers} from '';
+import {PoolService} from '';
+import {ModalHelpers} from '';
+
+interface pool {
+	pool_name: string;
+	pg_num: number;
+}
+export class PoolNewController {
+	private self = this;
+    private tier = this.tierList[0];
```

Comment:
Just keep only the declaration here, initialization should be done inside the constructor